server/db.js

```diff
@@ -1,4 +1,7 @@
 const mongoose = require("mongoose");
+const dotenv = require("dotenv");
+
+dotenv.config();
```

Comment:
We should call this inside the seed file rather than the db file because when we run the app the process is already available and don't need to call this again but the seed file is run once so best to put it there

Reply:
done